### DISCUSSION TITLE
Question on API Interface to LNM Webserver

### DISCUSSION THREAD

**Discussion by albar965** [albar965/littlenavmap#988] [Q&A] [answered]
@KOKAProduktion Can you answer the question below or help me to answer it? I'm not sure about the endpoints and cannot find the documentation.

https://www.avsim.com/forums/topic/629727-question-on-api-interface-to-lnm-webserver/

Alex

**Comment by KOKAProduktion** [comment, marked as answer]
Hi @albar965 ,

just downloaded the current 2.8.8 and was able to successfully perform the call in question via

http://localhost:8965/api/airport/info?ident=eddm

So i guess the issue the user is having is coming from the wrong parameter name (using "name" instead of "ident") and a missing error response from the API helping to identify the problem. (Raw response just says "Airport not found")

The proper usage and features of the API should be derived from the specification at [littlenavmap/web/webapi.yaml](https://github.com/albar965/littlenavmap/blob/master/web/webapi.yaml)

On a side note: The OpenLayers implementation under localhost:8965/ol seems broken at 2.8.8 as its JS bundle [littlenavmap/web/ol/index.bundle.js](https://github.com/albar965/littlenavmap/blob/master/web/ol/index.bundle.js) is not being delivered by the server (404). Not sure if that's of interest to that user...

I hope this helps! Otherwise let me know if i can do anything.

Cheers and thx!

**Reply by albar965** [reply]
Thanks. This helps a lot!

**Reply by hubha001** [reply]
Thanks a lot for your help here! I do not need any other modules. With your help i got the info I need and also understand now how to use the file webapi.yaml. Thanks a lot for all your work on this awesome program!
